cylc/flow/cfgspec/suite.py

```diff
@@ -774,8 +774,8 @@
 
                 The top level share and work directory location can be changed
                 (e.g. to a large data area) by a global config setting (see
-                :cylc:conf:`global.cylc[hosts][<hostname glob>]
-                work directory`).
+                :cylc:conf:`
+                flow.rc[platforms][<platform name>]work directory`).
```

Comment:
```suggestion
                global.cylc[platforms][<platform name>]work directory`).
```

Reply:
Also the last commit description should be `suite.rc -> flow.cylc`. Otherwise, can't find anything wrong 👍